Makefile

```diff
@@ -49,9 +49,6 @@ mandir = $(DESTDIR)$(PREFIX)/share/man/man1
 os = $(shell uname | sed 's/.*_NT.*/Windows/')
 os != uname | sed 's/.*_NT.*/Windows/'
 
-CPPFLAGS-os-Darwin = -I/opt/local/include
-LDFLAGS-os-Darwin = -L/opt/local/lib
-
```

Comment:
Probably the injection of /usr/local on FreeBSD/OpenBSD
can be updated the same way.

These were added back when we used third party libraries.
Now that we only use the C++ stdlib, there is probably no reasonable
scenario where we still need this.

Even if the compiler was installed to /usr/local/bin/cc,
it should still be able to find its associated standard library
(though I haven't tested that).

(In case it's not obvious to anyone, a possible workaround for this
would be `make CPPFLAGS-os-Darwin= LDFLAGS-os-Darwin=`)